pkg/color/color.go

```diff
@@ -0,0 +1,33 @@
+package color
```

Comment:
I added this package to use some coloring when printing failed actions, I don't know how helpful it is but I like color especially when adding more output.